lib/tasks/link-audit.rake

```diff
@@ -27,12 +27,17 @@ task link_audit: :environment do
     html = fetch(redirection.url)
 
     if html
-      next_link = "hotlinewebring.club/#{redirection.slug}/next"
-      prev_link = "hotlinewebring.club/#{redirection.slug}/previous"
+      # Surprisingly, this works just fine:
+      # <a href="https:&#x2F;&#x2F;hotlinewebring.club&#x2F;alexey&#x2F;next">next</a>
+      # Therefore, we should allow encoded slashes.
+      slash = %r{(/|&#x2F;)}
```

Comment:
Hmm, would this be clearer than what's there?

```ruby
slash = Regexp.union(['/', '&#x2f;'])
```

(I'm pretty sure the parentheses are unnecessary.)

Reply:
Yup! I like this better.

Reply:
Great, just made that change to use `Regexp.union` instead.